src/js/controller/settings/exportimage/PngExportController.js

```diff
@@ -30,12 +32,34 @@
   ns.PngExportController.prototype.onZipButtonClick_ = function () {
     var zip = new window.JSZip();
 
-    for (var i = 0 ; i < this.piskelController.getFrameCount() ; i++) {
-      var frame = this.piskelController.getFrameAt(i);
-      var canvas = this.getFrameAsCanvas_(frame);
-      var basename = this.pngFilePrefixInput.value;
-      var filename =  basename + (i + 1) + '.png';
-      zip.file(filename, pskl.utils.CanvasUtils.getBase64FromCanvas(canvas) + '\n', {base64: true});
+    function splittedExport() {
```

Comment:
We don't normally use named functions in Piskel, but try to attach functions to prototypes (except for some anonymous functions in forEach constructs, callbacks and so on ...).

To be consistent, we should do the same here.